deepecho/sequences.py

```diff
@@ -181,7 +182,8 @@ def assemble_sequences(
         groupby_columns = entity_columns[0] if len(entity_columns) == 1 else entity_columns
         for _, sequence in data.groupby(groupby_columns):
             sequence.drop(entity_columns, axis=1, inplace=True)
-            if context_columns:
+            missing_columns = [col for col in context_columns if col not in sequence.columns]
+            if context_columns and not missing_columns:
```

Comment:
should we check the other columns that are in sequence instead of skipping over? Or is the fake column the only one in context_columns?